include/IECore/IndexedIO.h

```diff
@@ -130,7 +131,7 @@ class IECORE_API IndexedIO : public RunTimeTyped
 		// singleton representing the root location (to be passed in the factory function)
 		static const EntryIDList rootPath;
 
-		typedef IndexedIOPtr (*CreatorFn)(const std::string &, const EntryIDList &, IndexedIO::OpenMode );
+		typedef IndexedIOPtr (*CreatorFn)(const std::string &, const EntryIDList &, IndexedIO::OpenMode, const CompoundData* options );
```

Comment:
should this one have a default value as well?

Reply:
Also, we don't do a great job of it elsewhere, but we should probably add some doxygen comments in this header describing the currently supported options.

Reply:
Don't think I can add an optional parameter to a typedef.

Reply:
Can you add :coffee: to my brain?